omero/sysadmins/limitations.rst

```diff
@@ -54,6 +54,10 @@ default. However, this process can be very resource-intensive, depending on the
 image as well as the image format and any data compression used, for example see
 `PixelData threads and pyramid generation issues <https://forum.image.sc/t/pixeldata-threads-and-pyramid-generation-issues/49794>`_.
 
+Furhter, OMERO never generates pyramids for large floating-point pixel type images.
+
+Also note that working with large floating-point images in OMERO can cause problems (see below).
```

Comment:
Is there a risk this communicates that OMERO cannot work with large floating-point images?

Reply:
fixed in https://github.com/ome/omero-documentation/pull/2256/commits/4456511ba1478ad7320577bc8f52474f81e7fb20